jib-core/src/main/java/com/google/cloud/tools/jib/docker/ImageTarball.java

```diff
@@ -16,17 +16,24 @@
 
 package com.google.cloud.tools.jib.docker;
```

Comment:
Also is `docker` the right package for this stuff? Should we have an oci package with an ImageTarball implementation?

Reply:
Hmm yeah I should probably move this to `image` or `tar`.